AstSemantics.md

```diff
@@ -1,14 +1,19 @@
 # Abstract Syntax Tree Semantics
 
-WebAssembly code is represented as an Abstract Syntax Tree (AST) where each node
-represents an expression. Each function body consists of a list of expressions.
-All expressions and operators are typed, with no implicit conversions or overloading rules.
+This document describes WebAssembly semantics. The description here is written
```

Comment:
why "*typed* stack machine"? if typed is important, why not "typed AST" as well?